examples/catalog-harness-iacm.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: demo-service-with-module
+  description: pipeline build and deploy
+  annotations:
+    harness.io/pipelines: |
```

Comment:
We can remove it from here as it is not relevant to IACM plugin

Reply:
done